app/views/planning_applications/assessment/ownership_certificates/_relevant_documents_accordion.html.erb

```diff
@@ -11,7 +11,7 @@
     <div class="govuk-accordion__section-content">
       <div class="govuk-!-padding-left-2 govuk-!-padding-right-2">
         <div class="scroll-docs">
-          <% @planning_application.documents.where("tags ?| array[:tags]", tags: Document::PLAN_TAGS).each do |document| %>
```

Comment:
Not even sure how this worked 🤯 

According to the [docs][1], the `?|` operator does the following

```
Do any of these array strings exist as top-level keys?
```

PostgreSQL must be treating the array of strings as keys 🙈 

[1]: https://www.postgresql.org/docs/9.5/functions-json.html#FUNCTIONS-JSONB-OP-TABLE